training_vue/public/task_vue_component.html

```diff
@@ -9,61 +9,131 @@
   <body>
     <div id="task-component">
       <div id="header">
-        <div id="system-name">Task Vue Component</div>
-        <div id="select-account">
-          アカウント：
-          <select>
-            <option>ダミーユーザーA</option>
-            <option>ダミーユーザーB</option>
-            <option>ダミーユーザーC</option>
-          </select>
-          <button>変更する</button>
-        </div>
+        <header-system-name></header-system-name>
+        <header-select-account v-bind:users="users" v-on:adduser="addUser"></header-select-account>
       </div>
 
       <div id="content">
-        <div>アカウントを選択してください</div>
-        <div class="sub-content">Content A</div>
-        <div class="sub-content">Content B</div>
-        <div class="sub-content">Content C</div>
+        <content-login-message v-bind:is-display="!isUserSelected"></content-login-message>
+        <content-sub-content title="Content A" v-bind:is-display="user.contentA"></content-sub-content>
+        <content-sub-content title="Content B" v-bind:is-display="user.contentB"></content-sub-content>
+        <content-sub-content title="Content C" v-bind:is-display="user.contentC"></content-sub-content>
```

Comment:
・データ自身がBooleanである
・コンポーネント自身の表示切り替えをするとき
のときは、直接v-showで表示非表示を切り替えてもOKです。

```suggestion
        <content-sub-content title="Content A" v-show="user.contentA"></content-sub-content>
        <content-sub-content title="Content B" v-show="user.contentB"></content-sub-content>
        <content-sub-content title="Content C" v-show="user.contentC"></content-sub-content>
```